ls1intum/Artemis
# Provide Assessment proposals to tutors for text exercises

### Checklist

### Description
This pull requests enables feedback suggestions on text submissions for tutors.
After an exercise was due, the submissions are split into text blocks and will be clustered into clusters of similar text blocks. ([see Concept here](https://brn.is/isee19))
Based on those clusters, Artemis will find and suggest Feedback based on previous submissions. (Fig. 2)

### Steps for Testing


1. Log in to ArTEMiS with ROLE_ADMIN
2. Navigate to Course Administration
3. Find a Text Exercises with Submissions and Edit
     i. The Due Date to a date a few minutes in the future.
     ii. Check Automatic Assessment (Fig. 1)
4. Assess the submissions and notice feedback suggestions as you move along.

### Screenshots
Fig. 1
<img width="300" alt="image" src="https://user-images.githubusercontent.com/6382716/61747969-6a84bc80-ad9f-11e9-9e6b-0aedc3863bd5.png">


Fig. 2
<img width="300" alt="image" src="https://user-images.githubusercontent.com/6382716/61747996-783a4200-ad9f-11e9-9b19-bc2be1f43f1b.png">
